packages/features/ee/organizations/pages/settings/appearance.tsx

```diff
@@ -27,7 +24,6 @@ type BrandColorsFormValues = {
 
 const OrgAppearanceView = ({
   currentOrg,
-  isAdminOrOwner,
```

Comment:
Only admin or owner should be able to change organization's appearance settings.

Reply:
We handle it in the profile section. Do we need to check each one individually?